docs/UserGuide.md

```diff
@@ -182,10 +186,18 @@ Examples:
 
 ### Marking all rooms as unclean: `markRoomsUnclean`
 
-Marks all room clean statuses of all guests as "no".
+This command functions to mark room clean statuses of all guests as "no".
+
+This command is typically used at the end or the start of the day when the hotel manager wishes to 
+update all the room clean statuses to "no" efficiently. This clean slate would allow the hotel 
+managers to keep abreast of the rooms that are cleaned or uncleaned for the day.
```

Comment:
This format seems different from the rest of the features, should we reformat the others then?

Reply:
Yes, reformatted some! Will do another around of reformatting when the rest commits their changes.